desc/objectives/_coils.py

```diff
@@ -602,6 +612,107 @@ def compute(self, params, constants=None):
         return out
 
 
+class CoilConvexity(_CoilObjective):
+    """Coil convexity.
```

Comment:
make it more clear that it is targeting that this objective is targeting to make the coil convex by making the that integral quantity minus 2pi = 0, that way the target of this objective being zero makes sense.

As is, it is ambiguous as to whether one should make the target 2pi or 0.